pylzy/lzy/storage/registry.py

```diff
@@ -6,39 +6,43 @@
 
 
 class DefaultStorageRegistry(StorageRegistry):
-    def __init__(self):
+    def __init__(self,
+                 default_storage_name: str,
+                 default_storage: Optional[Storage] = None):
         self.__credentials_map: Dict[str, Optional[Storage]] = defaultdict(
             lambda: None
         )
         self.__clients_map: Dict[str, Optional[AsyncStorageClient]] = defaultdict(
             lambda: None
         )
-
+        self.__default_name: str = default_storage_name
         self.__default_config: Optional[Storage] = None
-        self.__default_name: Optional[str] = None
         self.__default_client: Optional[AsyncStorageClient] = None
-
-    def register_storage(
-        self, name: str, storage: Storage, default: bool = False
-    ):
+        if default_storage is not None:
+            self.register_default_storage(default_storage)
+
+    def register_default_storage(self, default_storage: Storage):
```

Comment:
Why do we need an extra method for registering default storage?

Reply:
Unlike previous register_storage method, you can't set a name to default storage, so register_default_storage shouldn't accept a name as an argument. Now in old register_storage method we should check that given name is not a name of default storage. And now python client doesn't call register_storage, only register_default_storage, it makes sense.

Reply:
This is a bad idea. Assume, I firstly use default storage provided by lzy and it is registered with default storage name. This name is saved with whiteboards in whiteboard service. After that I change default storage with my own S3. New whiteboards are saved in my storage but with the same default storage name. This will lead to the situation when multiple whiteboards are saved in different storages but with the same storage name. It will be even impossible to load them.